src/components/Main/Main.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Todo } from "../../types/Todo";
+import cn from 'classnames';
+
+interface Props {
+  todos: Todo[];
+  loader: boolean;
+  tempId: number;
+}
+
+export const Main = ({ todos, loader, tempId }: Props) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.length > 0 ? (
+        todos.map((todo) => (
+          <div data-cy="Todo" className="todo" key={todo.id}>
```

Comment:
<img width="730" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/70220667/3e58231f-cd9b-4cd3-abd2-c083e321b027">
try to fix it